lambdas/dynamo_to_sns/dynamo_to_sns.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+"""
+Receives DynamoDB events and publishes the new image from the event to an SNS topic
+"""
+import json
+import os
+
+from sns_utils import publish_sns_message
+
+
+def main(event, _):
+    print(f'Received event:\n{event}')
+    stream_topic_map = json.loads(os.environ["STREAM_TOPIC_MAP"])
+    new_image = event['Records'][0]['dynamodb']['NewImage']
```

Comment:
At some point we might want to write nice `@property`-based wrapper for Lambda events that come from DynamoDB, but this'll do for now.